_tutorials/2016-06-23-copying-telemetry-policy-files-in-ios-xr-6-0-1.md

```diff
@@ -44,3 +44,51 @@ sftp&gt; <mark>quit</mark>
 </code>
 </pre>
 </div>
+
+If you wish to copy a policy file from the host Linux operating system to the IOS XR filesystem within the Vagrant box, then there is a two-step process. Firstly, a writeable directory must be created in disk0:
+
+<div class="highlighter-rouge">
+<pre class="highlight">
+<code>
+RP/0/RP0/CPU0:ios#<mark>run</mark>
+Tue Jun 28 06:40:16.346 UTC
+[xr-vm_node0_RP0_CPU0:~]$<mark>cd /disk0\:</mark>
+[xr-vm_node0_RP0_CPU0:~]$<mark>mkdir mypolicies</mark>
```

Comment:
Do you need to create a new directory under disk0: for a temporary file operation? Isn't disk0: enough?